src/components/GetStartedPopover/GetStartedPopover.tsx

```diff
@@ -1,106 +1,37 @@
 import { Flex, VisuallyHidden } from '@aws-amplify/ui-react';
-
 import { InternalLinkButton } from '@/components/InternalLinkButton';
-import {
-  IconAndroid,
-  IconAngular,
-  IconFlutter,
-  IconJS,
-  IconNext,
-  IconReact,
-  IconSwift,
-  IconVue
-} from '@/components/Icons';
-
 import { Popover } from '@/components/Popover';
+import { DEFAULT_PLATFORM, Platform } from '@/data/platforms';
+import { useIsGen1Page } from '@/utils/useIsGen1Page';
+import { UrlObject } from 'url';
 
-const getStartedHref = '/[platform]/start/getting-started/introduction/';
+export type GetStartedLinksType = {
+  title: string;
+  href: UrlObject;
+  icon: JSX.Element;
+  platform: Platform;
+};
 
-const getStartedLinks = [
-  {
-    title: 'React',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'react' }
-    },
-    icon: <IconReact />
-  },
-  {
-    title: 'JavaScript',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'javascript' }
-    },
-    icon: <IconJS />
-  },
-  {
-    title: 'Flutter',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'flutter' }
-    },
-    icon: <IconFlutter />
-  },
-  {
-    title: 'Swift',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'swift' }
-    },
-    icon: <IconSwift />
-  },
-  {
-    title: 'Android',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'android' }
-    },
-    icon: <IconAndroid />
-  },
-  {
-    title: 'React Native',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'react-native' }
-    },
-    icon: <IconReact />
-  },
-  {
-    title: 'Angular',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'angular' }
-    },
-    icon: <IconAngular />
-  },
-  {
-    title: 'Next.js',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'nextjs' }
-    },
-    icon: <IconNext />
-  },
-  {
-    title: 'Vue',
-    href: {
-      pathname: getStartedHref,
-      query: { platform: 'vue' }
-    },
-    icon: <IconVue />
-  }
-];
```

Comment:
Moved this into it's own utility function so that we could generate the gen1 and gen2 links.